26-Nov-2020 19:40:15.061 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
26-Nov-2020 19:40:15.061 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
26-Nov-2020 19:40:15.065 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@58dea0a5')
26-Nov-2020 20:11:40.487 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
26-Nov-2020 20:11:40.487 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
26-Nov-2020 20:11:40.490 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@52d10fb8')
26-Nov-2020 20:17:17.762 SEVERE [http-nio-8080-exec-6] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [Unable to compile class for JSP: 

An error occurred at line: [6] in the jsp file: [/login.jsp]
PrintWriter cannot be resolved to a type
3: <%!
4: Connection con;
5: PreparedStatement ps1;
6: PrintWriter out;
7: public void jspInit(){
8: try{
9:     Class.forName("com.mysql.cj.jdbc.Driver");


An error occurred at line: [34] in the jsp file: [/login.jsp]
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
31: out.println("<b>Invalid</b>");
32: else{
33:    session.putValue("email",emailUser);
34:    response.sendRedirect("..\..\HTML\Student\mainpage.html"); 
35: }
36: 
37: }


Stacktrace:] with root cause
	org.apache.jasper.JasperException: Unable to compile class for JSP: 

An error occurred at line: [6] in the jsp file: [/login.jsp]
PrintWriter cannot be resolved to a type
3: <%!
4: Connection con;
5: PreparedStatement ps1;
6: PrintWriter out;
7: public void jspInit(){
8: try{
9:     Class.forName("com.mysql.cj.jdbc.Driver");


An error occurred at line: [34] in the jsp file: [/login.jsp]
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
31: out.println("<b>Invalid</b>");
32: else{
33:    session.putValue("email",emailUser);
34:    response.sendRedirect("..\..\HTML\Student\mainpage.html"); 
35: }
36: 
37: }


Stacktrace:
		at org.apache.jasper.compiler.DefaultErrorHandler.javacError(DefaultErrorHandler.java:103)
		at org.apache.jasper.compiler.ErrorDispatcher.javacError(ErrorDispatcher.java:213)
		at org.apache.jasper.compiler.JDTCompiler.generateClass(JDTCompiler.java:489)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:392)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:362)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:346)
		at org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:605)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:400)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:19:02.647 SEVERE [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [Unable to compile class for JSP: 

An error occurred at line: [6] in the jsp file: [/login.jsp]
PrintWriter cannot be resolved to a type
3: <%!
4: Connection con;
5: PreparedStatement ps1;
6: PrintWriter out;
7: public void jspInit(){
8: try{
9:     Class.forName("com.mysql.cj.jdbc.Driver");


An error occurred at line: [34] in the jsp file: [/login.jsp]
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
31: out.println("<b>Invalid</b>");
32: else{
33:    session.putValue("email",emailUser);
34:    response.sendRedirect("..\..\HTML\Student\mainpage.html"); 
35: }
36: 
37: }


Stacktrace:] with root cause
	org.apache.jasper.JasperException: Unable to compile class for JSP: 

An error occurred at line: [6] in the jsp file: [/login.jsp]
PrintWriter cannot be resolved to a type
3: <%!
4: Connection con;
5: PreparedStatement ps1;
6: PrintWriter out;
7: public void jspInit(){
8: try{
9:     Class.forName("com.mysql.cj.jdbc.Driver");


An error occurred at line: [34] in the jsp file: [/login.jsp]
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
31: out.println("<b>Invalid</b>");
32: else{
33:    session.putValue("email",emailUser);
34:    response.sendRedirect("..\..\HTML\Student\mainpage.html"); 
35: }
36: 
37: }


Stacktrace:
		at org.apache.jasper.compiler.DefaultErrorHandler.javacError(DefaultErrorHandler.java:103)
		at org.apache.jasper.compiler.ErrorDispatcher.javacError(ErrorDispatcher.java:213)
		at org.apache.jasper.compiler.JDTCompiler.generateClass(JDTCompiler.java:489)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:392)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:362)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:346)
		at org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:605)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:400)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:20:42.684 SEVERE [http-nio-8080-exec-10] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [Unable to compile class for JSP: 

An error occurred at line: [20] in the jsp file: [/login.jsp]
request cannot be resolved
17: Statement stmt=con.createStatement();
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");


An error occurred at line: [21] in the jsp file: [/login.jsp]
request cannot be resolved
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");
24: ResultSet rs = ps1.executeQuery();


An error occurred at line: [30] in the jsp file: [/login.jsp]
out cannot be resolved
27: c = rs.getInt(1);
28: 
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 


An error occurred at line: [32] in the jsp file: [/login.jsp]
session cannot be resolved
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 


An error occurred at line: [33] in the jsp file: [/login.jsp]
response cannot be resolved
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 
36: }


Stacktrace:] with root cause
	org.apache.jasper.JasperException: Unable to compile class for JSP: 

An error occurred at line: [20] in the jsp file: [/login.jsp]
request cannot be resolved
17: Statement stmt=con.createStatement();
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");


An error occurred at line: [21] in the jsp file: [/login.jsp]
request cannot be resolved
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");
24: ResultSet rs = ps1.executeQuery();


An error occurred at line: [30] in the jsp file: [/login.jsp]
out cannot be resolved
27: c = rs.getInt(1);
28: 
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 


An error occurred at line: [32] in the jsp file: [/login.jsp]
session cannot be resolved
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 


An error occurred at line: [33] in the jsp file: [/login.jsp]
response cannot be resolved
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 
36: }


Stacktrace:
		at org.apache.jasper.compiler.DefaultErrorHandler.javacError(DefaultErrorHandler.java:103)
		at org.apache.jasper.compiler.ErrorDispatcher.javacError(ErrorDispatcher.java:213)
		at org.apache.jasper.compiler.JDTCompiler.generateClass(JDTCompiler.java:489)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:392)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:362)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:346)
		at org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:605)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:400)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:24:57.487 SEVERE [http-nio-8080-exec-3] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [Unable to compile class for JSP: 

An error occurred at line: [20] in the jsp file: [/login.jsp]
request cannot be resolved
17: Statement stmt=con.createStatement();
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");


An error occurred at line: [21] in the jsp file: [/login.jsp]
request cannot be resolved
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");
24: ResultSet rs = ps1.executeQuery();


An error occurred at line: [30] in the jsp file: [/login.jsp]
out cannot be resolved
27: c = rs.getInt(1);
28: 
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 


An error occurred at line: [32] in the jsp file: [/login.jsp]
session cannot be resolved
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 


An error occurred at line: [33] in the jsp file: [/login.jsp]
response cannot be resolved
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 
36: }


Stacktrace:] with root cause
	org.apache.jasper.JasperException: Unable to compile class for JSP: 

An error occurred at line: [20] in the jsp file: [/login.jsp]
request cannot be resolved
17: Statement stmt=con.createStatement();
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");


An error occurred at line: [21] in the jsp file: [/login.jsp]
request cannot be resolved
18: stmt.executeQuery(query);
19: 
20: String emailUser = request.getParameter("email");
21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");
24: ResultSet rs = ps1.executeQuery();


An error occurred at line: [30] in the jsp file: [/login.jsp]
out cannot be resolved
27: c = rs.getInt(1);
28: 
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 


An error occurred at line: [32] in the jsp file: [/login.jsp]
session cannot be resolved
29: if(c==0)
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 


An error occurred at line: [33] in the jsp file: [/login.jsp]
response cannot be resolved
30: out.println("<b>Invalid</b>");
31: else{
32:    session.putValue("email",emailUser);
33:    response.sendRedirect("..\\..\\HTML\\Student\\mainpage.html"); 
34: }
35: 
36: }


Stacktrace:
		at org.apache.jasper.compiler.DefaultErrorHandler.javacError(DefaultErrorHandler.java:103)
		at org.apache.jasper.compiler.ErrorDispatcher.javacError(ErrorDispatcher.java:213)
		at org.apache.jasper.compiler.JDTCompiler.generateClass(JDTCompiler.java:489)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:392)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:362)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:346)
		at org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:605)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:400)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:28:49.071 SEVERE [http-nio-8080-exec-4] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [Unable to compile class for JSP: 

An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
void[] is an invalid type
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Syntax error on token "jspInit", AnnotationName expected after this token
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Illegal modifier for the variable jspInit; only final is permitted
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Syntax error, insert "[ ]" to complete Dimension
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
void[] is an invalid type
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Syntax error on token "jspDestroy", AnnotationName expected after this token
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Illegal modifier for the variable jspDestroy; only final is permitted
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Syntax error, insert "[ ]" to complete Dimension
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


Stacktrace:] with root cause
	org.apache.jasper.JasperException: Unable to compile class for JSP: 

An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
void[] is an invalid type
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Syntax error on token "jspInit", AnnotationName expected after this token
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Illegal modifier for the variable jspInit; only final is permitted
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Syntax error, insert "[ ]" to complete Dimension
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [6] in the jsp file: [/webapp/login.jsp]
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
3: <%
4: Connection con;
5: PreparedStatement ps1;
6: public void jspInit(){
7: try{
8:     Class.forName("com.mysql.cj.jdbc.Driver");
9: }


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
void[] is an invalid type
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Syntax error on token "jspDestroy", AnnotationName expected after this token
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Illegal modifier for the variable jspDestroy; only final is permitted
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Syntax error, insert "[ ]" to complete Dimension
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


An error occurred at line: [41] in the jsp file: [/webapp/login.jsp]
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
38: %>
39: 
40: <%
41:     public void jspDestroy()
42:     {
43:         try
44:         {


Stacktrace:
		at org.apache.jasper.compiler.DefaultErrorHandler.javacError(DefaultErrorHandler.java:103)
		at org.apache.jasper.compiler.ErrorDispatcher.javacError(ErrorDispatcher.java:213)
		at org.apache.jasper.compiler.JDTCompiler.generateClass(JDTCompiler.java:489)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:392)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:362)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:346)
		at org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:605)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:400)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:31:57.668 SEVERE [http-nio-8080-exec-6] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [An exception occurred processing [/webapp/login.jsp] at line [15]

12: }
13: System.out.println("Class found");
14: 
15: con=DriverManager.getConnection("jdbc:mysql://localhost:3306/ncp","root","RC7pA9vB");
16: String query="USE NCP";
17: Statement stmt=con.createStatement();
18: stmt.executeQuery(query);


Stacktrace:] with root cause
	java.sql.SQLException: No suitable driver found for jdbc:mysql://localhost:3306/ncp
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:702)
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
		at org.apache.jsp.webapp.login_jsp._jspService(login_jsp.java:134)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:35:18.495 SEVERE [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [An exception occurred processing [/webapp/login.jsp] at line [15]

12: }
13: System.out.println("Class found");
14: 
15: con=DriverManager.getConnection("jdbc:mysql://localhost:3306/ncp","root","RC7pA9vB");
16: String query="USE NCP";
17: Statement stmt=con.createStatement();
18: stmt.executeQuery(query);


Stacktrace:] with root cause
	java.sql.SQLException: No suitable driver found for jdbc:mysql://localhost:3306/ncp
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:702)
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
		at org.apache.jsp.webapp.login_jsp._jspService(login_jsp.java:134)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:37:40.408 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
26-Nov-2020 20:37:40.408 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
26-Nov-2020 20:37:40.413 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@470d183')
26-Nov-2020 20:37:43.667 SEVERE [http-nio-8080-exec-10] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [An exception occurred processing [/webapp/login.jsp] at line [15]

12: }
13: System.out.println("Class found");
14: 
15: con=DriverManager.getConnection("jdbc:mysql://localhost:3306/ncp","root","RC7pA9vB");
16: String query="USE NCP";
17: Statement stmt=con.createStatement();
18: stmt.executeQuery(query);


Stacktrace:] with root cause
	java.sql.SQLException: No suitable driver found for jdbc:mysql://localhost:3306/ncp
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:702)
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
		at org.apache.jsp.webapp.login_jsp._jspService(login_jsp.java:134)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:42:59.433 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
26-Nov-2020 20:42:59.433 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
26-Nov-2020 20:42:59.436 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@2a53f215')
26-Nov-2020 20:43:14.094 SEVERE [http-nio-8080-exec-2] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [An exception occurred processing [webapp/login.jsp] at line [24]

21: String pass = request.getParameter("pass");
22: 
23: ps1 = con.prepareStatement("select count(*) from Users where EmailID = ? and PasswordVal = ?");
24: ResultSet rs = ps1.executeQuery();
25: int c = 0;
26: if(rs.next())
27: c = rs.getInt(1);


Stacktrace:] with root cause
	java.sql.SQLException: No value specified for parameter 1
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
		at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
		at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1020)
		at org.apache.jsp.webapp.login_jsp._jspService(login_jsp.java:143)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 20:46:30.337 SEVERE [http-nio-8080-exec-3] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [] threw exception [Unable to compile class for JSP: 

An error occurred at line: [35] in the jsp file: [/webapp/login.jsp]
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
32: out.println("<b>Invalid</b>");
33: else{
34:    session.putValue("email",emailUser);
35:    response.sendRedirect("D:\\4th Year\\NCP\NCPProject\\HTML\\Student\\mainpage.html"); 
36: }
37: 
38: 


Stacktrace:] with root cause
	org.apache.jasper.JasperException: Unable to compile class for JSP: 

An error occurred at line: [35] in the jsp file: [/webapp/login.jsp]
Invalid escape sequence (valid ones are  \b  \t  \n  \f  \r  \"  \'  \\ )
32: out.println("<b>Invalid</b>");
33: else{
34:    session.putValue("email",emailUser);
35:    response.sendRedirect("D:\\4th Year\\NCP\NCPProject\\HTML\\Student\\mainpage.html"); 
36: }
37: 
38: 


Stacktrace:
		at org.apache.jasper.compiler.DefaultErrorHandler.javacError(DefaultErrorHandler.java:103)
		at org.apache.jasper.compiler.ErrorDispatcher.javacError(ErrorDispatcher.java:213)
		at org.apache.jasper.compiler.JDTCompiler.generateClass(JDTCompiler.java:489)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:392)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:362)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:346)
		at org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:605)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:400)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:880)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1601)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
26-Nov-2020 22:13:45.218 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
26-Nov-2020 22:13:45.218 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
26-Nov-2020 22:13:45.223 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@3ce443f9')
26-Nov-2020 22:22:11.494 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
26-Nov-2020 22:22:11.495 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
26-Nov-2020 22:22:11.499 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@55651434')
